contrib/firmware_packager/simple_client.py

```diff
@@ -110,6 +111,16 @@ def device_changed(client, device, progress):  # pylint: disable=unused-argument
     progress.device_changed(device.get_name())
 
 
+def modify_config(client, key, value):
+    """Use fwupd client to modify daemon configuration value"""
+    try:
+        client.modify_config(key, value, None)
```

Comment:
I think you should emit something to stdout that this happened.  It shouldn't be FULLY invisible to the user that they're waiving security to do this.